### PR TITLE
Add CheckboxRadio component

### PR DESCRIPTION
**Pull request by skjnldsv**
- Uses the Material design icons
- Supports simple, sets and single-set checkboxes + radio (see docs)
- Supports indeterminate state
- Very straightforward! :rocket: 

A few examples:
image
image
image

### CODE REVIEW

**Comment by jancborchardt** [comment]
Wooooooah very nice! :) Sooo good to have this standardized.

Some design feedback:
- Each of the rows should be 44px height to conform to the min clickable/tappable area. The resulting whitespace will also look nice as  separation.
- The radio set using checkmarks looks a bit off. Normally it’s a dot in a circle, like so https://material.io/develop/web/components/selection-controls/radio-buttons (and like we do currently already)
- What is the "Standard checkbox set" with a square inside the square? If that is the indeterminate setting – we should keep it how it looks like for us currently (with the dash/minus in there) since that is also kind of a standard: https://material.io/components/checkboxes#behavior

**Review by jancborchardt — APPROVED** [review]
Nice, works very well now, looks great and is accessible. :+1: design-wise!

File: src/components/CheckboxRadio/CheckboxRadio.vue [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+		 * Unique id attribute of the input
+		 */
+		id: {
+			type: String,
+			default: () => 'checkbox-radio-' + GenRandomId(),
+			validator: id => id.trim() !== '',
+		},
+
+		/**
+		 * Input name. Especially required for radios set
```
Comment (ChristophWurst) [inline src/components/CheckboxRadio/CheckboxRadio.vue:156]: "especially required" -> is it required or not?

File: src/components/CheckboxRadio/CheckboxRadio.vue [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+		/**
+		 * Disabled state
+		 */
+		disabled: {
+			type: Boolean,
+			default: false,
+		},
+
+		/**
+		 * Disabled state
```
Comment (ChristophWurst) [inline src/components/CheckboxRadio/CheckboxRadio.vue:197]: :thinking:

File: src/components/CheckboxRadio/CheckboxRadio.vue (hunk 1 above)
Comment (skjnldsv) [reply src/components/CheckboxRadio/CheckboxRadio.vue:156]: You can't do conditional required.
It is for radio, unless you want a single radio, which you can, buuuuut doesn't make any ses=nse xD

Comment (ChristophWurst) [reply src/components/CheckboxRadio/CheckboxRadio.vue:156]: then say "required for radio, optional for checkbox"?

Comment (skjnldsv) [reply src/components/CheckboxRadio/CheckboxRadio.vue:156]: :+1:

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**